lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop check for usages of not called on a method when an
+      # inverse of that method can be used instead
+      #
+      # @example
+      #   # bad
+      #   !foo.none?
+      #   !foo.any? { |f| f.even? }
+      #   !foo.blank?
+      #
+      #   # good
+      #   foo.none?
+      #   foo.blank?
+      #   foo.any? { |f| f.even? }
+      class InverseMethods < Cop
+        MSG = 'Use `%s` instead of `!.%s`.'.freeze
+
+        RUBY_METHODS = {
+          any?: :none?
```

Comment:
> It would be awesome if we could do something similar with select and reject, but I feel like the that might get complicated. They also don't follow the current pattern.

Another candidate then would be `grep_v` and `grep`